omod/src/main/java/org/openmrs/module/coreapps/page/controller/clinicianfacing/PatientPageController.java

```diff
@@ -14,12 +14,12 @@
 package org.openmrs.module.coreapps.page.controller.clinicianfacing;
 
 import org.apache.commons.lang3.StringUtils;
-import org.openmrs.Location;
-import org.openmrs.Patient;
+import org.openmrs.*;
```

Comment:
We do not encourage using wild card imports as per our convention: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports